webcompat/form.py

```diff
@@ -274,6 +291,11 @@ def build_formdata(form_object):
 {steps_reproduce}
 
 """.format(**formdata)
+    # Append details info, if any.
+    details = form_object.get('details')
+    if details:
+        body += u'\n\n**Details**\n{details}'.format(
+            details=get_details(details))
```

Comment:
Just curious here. Why not adding

```
<details>
<summary>Browser Configuration</summary>
<ul>
    <li>blah</li>
    <li>blah 2</li>
    <li>blah 3</li>
</ul>
</details>
```

Reply:
We can try this, no real reason. I'm not sure how the markdown parser would handle it, but hopefully nothing surprising would happen.